src/main/java/lesson9Refactoring2/Engine.java

```diff
@@ -0,0 +1,38 @@
+package lesson9Refactoring2;
+
+public class Engine {
+
+    private boolean engineStarted = false;
+    private final double size;
```

Comment:
Skąd to?

Reply:
dodałem kilka dodatkowych funkcji od siebie, by rozwinąć trochę kod